src/contributor-guide/contributing.md

```diff
@@ -95,6 +95,16 @@ The following list consists of questions you can ask to help determine the prope
 
 The number of 'Yes' answers should help you to determine the severity.
 
+### Pull request risk assessment:
+
+The risk label highlights the risk that the suggested changes may bring to the platform, it helps us to process pull requests, to decide on which version the pull requests should be delivered, it highlights pull requests that may need to be reviewed by other people, and it helps to decide whether a pull request should be approved or not depending on its risk.
+
+| Risk | Description |
+| ------------- | ------------- |
+| High | A pull request that makes changes on the framework or changes that will affect other areas. |
+| Medium | A pull request that makes changes that may affect other areas or it makes considerable changes on specific area. |
+| Low | A pull request that it's changes likely won't affect other areas. |
```

Comment:
Should avoid contractions in technical documents - people with English as a second language may not understand.
```suggestion
| Low | A pull request that will probably not affect other areas. |
```